### PR TITLE
MODPUBSUB-65:Extend documentation for mod-pubsub.

### PR DESCRIPTION
**Pull request by VRohach**
1. 'README.md'- file was updated.

### CODE REVIEW

**Comment by VRohach** [comment]
Feel free to propose other updates to this PR.

**Comment by VRohach** [comment]
Guys, feel free to review this PR

File: README.md [hunk 1]
```diff
@@ -240,6 +240,200 @@ To publish an event `PubSubClientUtils` class provides `sendEventMessage` method
...
+}
+```
+Keep in mind, that subscriber module should return response to the "mod-pubsub" before all business-logic. Main reasons for it:
+- Extra connection to the module should be closed for best performance;
+- There is no need for waiting response from subscriber's logic because this is not "mod-pubsub" responsibility. It should only deliver event to this module.
+
+As result, we can just trigger "/publisher/publish" endpoint, and publisher module will sent event to the "mod-pubsub". After that, "mod-pubsub" will deliver it to the subscriber module. And it will log this event to the console.
+
+####Permissions
+#####"mod-pubsub" permissions workflow:
```
Comment (OleksiiKuzminov) [inline README.md]: plz add information about subscribers module description permissions and were we declaring permissions in mod-pub-sub

Comment (VRohach) [reply README.md]: Done. Could you please review ?

**Pull request merged**